### ISSUE TITLE
sequence length is longer than the specified maximum

### ISSUE THREAD

**Issue by DecentMakeover** [zalandoresearch/flair#836]
This is my current script 

corpus: Corpus = NLPTaskDataFetcher.load_column_corpus(data_folder, columns,
                                                                  train_file=train_path,
                                                                  test_file=test_path,
                                                                  dev_file=test_path)
    # 2. what tag do we want to predict?
    tag_type = 'ner'
    import pdb;pdb.set_trace()

    # 3. make the tag dictionary from the corpus
    tag_dictionary = corpus.make_tag_dictionary(tag_type=tag_type)
    print(tag_dictionary.idx2item)

    # 4. initialize embeddings
    embedding_types: List[TokenEmbeddings] = [
        # ELMoEmbeddings('elmo-small'),
        # WordEmbeddings('glove`'),
        BertEmbeddings(),
        # comment in this line to use character embeddings
        # CharacterEmbeddings(),

        # comment in these lines to use flair embeddings
        # FlairEmbeddings('news-forward'),
        # FlairEmbeddings('news-backward'),
    ]

    embeddings: StackedEmbeddings = StackedEmbeddings(embeddings=embedding_types)

    # 5. initialize sequence tagger
    from flair.models import SequenceTagger

    tagger: SequenceTagger = SequenceTagger(hidden_size=256,
                                            embeddings=embeddings,
                                            tag_dictionary=tag_dictionary,
                                            tag_type=tag_type,
                                            use_crf=True)

    # 6. initialize trainer
    from flair.trainers import ModelTrainer

    trainer: ModelTrainer = ModelTrainer(tagger, corpus)

    # 7. start training
    trainer.train(model_info,
                  learning_rate=0.1,
                  mini_batch_size=10,
                  max_epochs=150)

    # 8. plot training curves (optional)
    from flair.visual.training_curves import Plotter
    plotter = Plotter()
    plotter.plot_training_curves(os.path.join(model_info,'loss.tsv'))
    plotter.plot_weights(os.path.join(model_info,'weights.txt'))

It starts training but after a while 

19-06-25 00:29:47,128 epoch 1 - iter 396/1322 - loss 20.69170048
    2019-06-25 00:37:21,113 epoch 1 - iter 528/1322 - loss 18.01724663
    2019-06-25 00:44:57,307 epoch 1 - iter 660/1322 - loss 16.23165289
    2019-06-25 00:52:46,592 epoch 1 - iter 792/1322 - loss 14.92485621
    2019-06-25 01:00:06,629 epoch 1 - iter 924/1322 - loss 13.84436923
    2019-06-25 01:07:51,824 epoch 1 - iter 1056/1322 - loss 12.99097852
    2019-06-25 01:15:29,695 epoch 1 - iter 1188/1322 - loss 12.25934242
    2019-06-25 01:23:16,301 epoch 1 - iter 1320/1322 - loss 11.66648230
    2019-06-25 01:23:18,473 ----------------------------------------------------------------------------------------------------
    2019-06-25 01:23:18,473 EPOCH 1 done: loss 11.6611 - lr 0.1000
    2019-06-25 01:23:47,627 Token indices sequence length is longer than the specified maximum  sequence length for this BERT model (576 > 512). Running this sequence through BERT will result in indexing errors
    /pytorch/aten/src/THC/THCTensorIndex.cu:362: void indexSelectLargeIndex(TensorInfo<T, IndexType>, TensorInfo<T, IndexType>, TensorInfo<long, IndexType>, int, int, IndexType, IndexType, long) [with T = float, Ind$
    xType = unsigned int, DstDim = 2, SrcDim = 2, IdxDim = -2, IndexIsMajor = true]: block: [162,0,0], thread: [32,0,0] Assertion `srcIndex < srcSelectDimSize` failed.

Training till the moment it gets terminated takes a long time, how do i check that the sequence length matches what bert expects, Thanks in advance.

**Comment by alanakbik** [comment]
Hello @DecentMakeover this question is answered in #831, so I'll mark as duplicate and close. Feel free to reopen if you have more questions.

**Comment by DecentMakeover** [comment]
Okay thanks , ill check.
